moldesign/widgets/computeconfig.py

```diff
@@ -71,6 +73,55 @@ def _makelink(url, text):
                                                                                   text=text)
 
 
+class ChangeLog(ipy.Box):
```

Comment:
This only checks and nags users to update - for right now, anyway.

(Also, the update check is synchronous, which is bad especially if the user isn't connected to the internet. This may be one of those things that should get moved to javascript)